modules/animationqt/src/animationeditordockwidgetqt.cpp

```diff
@@ -312,18 +313,47 @@ AnimationEditorDockWidgetQt::AnimationEditorDockWidgetQt(
                 [&]() { controller_.eval(controller_.getCurrentTime(), Seconds(0.0)); });
     }
 
+    {
+        // For presentation, e.g, a clicker (Key_PageUp)
+        auto* prevControlKeyframe = mainWindow_->addAction("Prev Control Keyframe");
```

Comment:
Should this not also be in the menu to make it more discoverable?

Reply:
We can have items in the menu without having it in the toolbar. I would not put it in the toolbar either. But I'm thinking as a text entry in the menu.